config/application.rb

```diff
@@ -15,7 +15,10 @@ class Application < Rails::Application
     end
 
     # Use Redis as cache store
-    config.cache_store = :redis_store, ENV['REDIS_URL'] if ENV['REDIS_URL'].present?
+    config.cache_store = :redis_store, "#{ENV['REDIS_URL']}/cache" if ENV['REDIS_URL'].present?
+
+    # Use Redis for session store
+    config.session_store :redis_store, servers: ["#{ENV['REDIS_URL']}/session"]
```

Comment:
Is it worth guarding against missing REDIS_URL here as well?

Reply:
Yes very good point